codeclimate-wrapper

```diff
@@ -41,17 +41,34 @@ else
     && invalid_docker_host "$DOCKER_HOST"
 fi
 
+CC_XDG_CONFIG_HOME=${XDG_CONFIG_HOME:-~/.config/}
+CC_XDG_CONFIG="${CC_XDG_CONFIG_HOME}/codeclimate/config.yml"
+CC_XDG_CONFIG_DIR=$(dirname "${CC_XDG_CONFIG}")
+[ -d "$CC_XDG_CONFIG_DIR" ] || mkdir -p "$CC_XDG_CONFIG_DIR"
+[ -f "$CC_XDG_CONFIG" ] || ( echo "---" > "$CC_XDG_CONFIG" )
```

Comment:
These actions are idempotent (if you replace `echo` with `touch`), so I would just always do it. You can also DRY it up with below via a loop:

```sh
for file in "$CC_CONFIG" "$CC_CACHE"; do
  mkdir -p "$(dirname "$file")"
  touch "$file"
done
```